include/jwt-cpp/jwt.h

```diff
@@ -2668,6 +2668,162 @@ namespace jwt {
 			}
 		}
 	};
+	
+	template<typename json_traits>
+	class jwks {
+		using basic_claim_t = basic_claim<json_traits>;
+		template<typename> friend class jwks_keys;
+
+	public:
+		JWT_CLAIM_EXPLICIT jwks(const typename json_traits::string_type& jwks_token) {
+			auto parse_claims = [](const typename json_traits::string_type& str) {
```

Comment:
this block seems to be copy pasted from the original

📓 for myself to look into a small refactor!

Reply:
yes, and it costs an extra move, good point to refactor.